crates/biome_js_analyze/tests/specs/a11y/useSemanticElements/invalid.jsx

```diff
@@ -7,14 +7,14 @@
     <div role="separator" ></div>
     <div role="button" ></div>
     <div role="article" ></div>
-    <div role="dialog" ></div>
+
     <div role="cell" ></div>
     <div role="columnheader" ></div>
-    <div role="definition" ></div>
+
     <div role="figure" ></div>
     <div role="form" ></div>
-    <div role="graphics-document" ></div>
-    <div role="graphics-object" ></div>
```

Comment:
These roles are part of [WAI-ARIA Graphics Module](https://w3c.github.io/graphics-aria/). I will add them back in a future PR.